.dotnet/example_code/DynamoDB/TryDax/03-GetItem-Test.cs

```diff
@@ -25,26 +25,19 @@
 using Amazon.Runtime;
 using Amazon.DAX;
 using Amazon.DynamoDBv2.Model;
-using System.Collections.Generic;
-using System;
-using Amazon.DynamoDBv2;
-using Amazon;
 
 namespace ClientTest
 {
     class Program
     {
         static void Main(string[] args)
         {
+            string endpointUri = args[0];
+            Console.WriteLine("Using DAX client - endpointUri=" + endpointUri);
```

Comment:
This statement should also use interpolated strings. For example:

Console.WriteLine($"Using DAX client - endpointUri={endpointUri}");

You wouldn't do this with strings that are to be translated to another language, but in this case and throughout the code, you should use interpolated strings instead of concatenating them.

Reply:
Revised as per discussion to match approved samples in TryDax.zip.